apps/examples/testcase/ta_tc/messaging/utc/utc_messaging_recv.c

```diff
@@ -152,6 +156,7 @@ static void nonblock_callback(msg_reply_type_t msg_type, msg_recv_buf_t *recv_da
 {
 	int ret;
 	msg_send_data_t reply;
+	reply.msg = NULL;
```

Comment:
If line 161 is false, line 181 tries to free null. I think you should move line 181 into if case.

Reply:
I agree with your opinion.
I will move line 181 into end of if case